api/db-cdr/generate-cdr/libproject/devstart.rb

```diff
@@ -50,7 +50,7 @@ def get_config(env)
   return JSON.parse(File.read("../../config/" + ENVIRONMENTS[env][:config_json]))
 end
 
-def get_auth_domain_group(project)
+def get_auth_domain_group_email(project)
```

Comment:
Strictly a rename.  We use the group *name* elsewhere, so this is only to clarify that this is different.